modules/nf-core/smncopynumbercaller/main.nf

```diff
@@ -34,19 +34,19 @@ process SMNCOPYNUMBERCALLER {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        SMNCopyNumberCaller: $VERSION
+        SMNCopyNumberCaller: "1.1.2"
```

Comment:
The use of `$VERSION` is how all modules do it and the guidelines show that it should be like this: https://nf-co.re/docs/contributing/modules#general paragraph 4, the second part